packages/block-library/src/image/edit.js

```diff
@@ -438,11 +463,12 @@ class ImageEdit extends Component {
 			);
 		}
 
-		const classes = classnames( className, {
+		const classNameWithoutSize = className.replace( /(\s+?)size-(.*?)(\s+?)/, '' );
+		const classes = classnames( classNameWithoutSize.value, {
```

Comment:
I think this is wrong, a leftover from trying to use a tokenizer. Will fix

Reply:
> trying to use a tokenizer

If still needed, or for future reference, there is a module which exists already which serves this purpose quite well:

https://github.com/WordPress/gutenberg/tree/master/packages/token-list